scripts/vote_2023_06_20_2.py

```diff
@@ -0,0 +1,63 @@
+"""
+Voting
+
+1) Stake all Treasury ETH in Lido
+"""
+
+import time
+
+from typing import Dict, Tuple, Optional
+
+from brownie.network.transaction import TransactionReceipt
+from utils.agent import agent_execute
+from brownie import ZERO_ADDRESS
+
+from utils.voting import bake_vote_items, confirm_vote_script, create_vote
+
+from utils.config import (
+    get_deployer_account,
+    get_is_live,
+    contracts,
+    get_priority_fee,
+)
+
+
+def start_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[TransactionReceipt]]:
+    """Prepare and run voting."""
+
+    current_agent_balance = contracts.agent.balance()
+
+    submit_calldata = contracts.lido.submit.encode_input(ZERO_ADDRESS)
+    print(
+            contracts.lido.address,
+            current_agent_balance,
+            'Stake all Treasury ETH in Lido'
+        )
+    call_script_items = [
+        agent_execute(
```

Comment:
Why the usual "agent.forward" doesn't fit?

Reply:
foward has no ability to pass eth value